src/middleware/devtools.ts

```diff
@@ -1,5 +1,39 @@
 import { GetState, PartialState, SetState, State, StoreApi } from '../vanilla'
 
+type DevtoolsOptions = {
+  name?: string
+  anonymousActionType?: string
+  serialize?: {
+    options:
+      | boolean
+      | {
+          date?: boolean
+          regex?: boolean
+          undefined?: boolean
+          nan?: boolean
+          infinity?: boolean
+          error?: boolean
+          symbol?: boolean
+          map?: boolean
+          set?: boolean
+        }
+  }
+  features?: {
+    pause?: boolean
+    lock?: boolean
+    persist?: boolean
+    export?: boolean | 'custom'
+    import?: boolean | 'custom'
+    jump?: boolean
+    skip?: boolean
+    reorder?: boolean
+    dispatch?: boolean
+    test?: boolean
+  }
+  trace?: boolean | (() => string)
```

Comment:
Let's remove `(() => string)` if it's not working. (I assume we need to change our implementation to support it?)